internal/pkg/debug/render/dot.go

```diff
@@ -46,53 +45,59 @@ func (r *renderer) Render() string {
 }
 
 func (r *renderer) init() {
-	_, _ = r.WriteString("digraph {\n")
```

Comment:
I have removed this here and elsewhere. It cannot actually fail, since we are writing to a `strings.Builder`, so explicitly ignoring the `int, error` return values is just noise that makes the code harder to read.